Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin(jenkinsVersions: [null, '2.32.3'])
```

Comment:
cc @jglick esp. for this change, given you were the one adding it.

Reply:
fine